File: programmers/js/level01/12910.js
// 나누어 떨어지는 숫자 배열
function solution(arr, divisor) {
  const filterArr = arr.filter(num => !(num % divisor));
  
  return filterArr.length > 0 ? filterArr.sort((a, b) => a - b) : [-1];
}

File: programmers/java/level01/12910.java
// 나누어 떨어지는 숫자 배열

import java.util.Arrays;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        int[] result = Arrays.stream(arr)
            .filter(num -> num % divisor == 0)
            .sorted()
            .toArray();
        
        return result.length > 0 ? result : new int[]{ -1 };
    }
}